README.md

```diff
@@ -1,207 +1,249 @@
+| dim_order  | string    | **REQUIRED.** How the above dimensions are ordered within the `shape`. `bhw`, `bchw`, `bthw`, `btchw` are valid orderings where `b`=batch, `c`=channel, `t`=time, `h`=height, w=width.                                                                                                                                                         |     |
+| data_type  | enum      | **REQUIRED.** The data type of values in the n-dimensional array. For model inputs, this should be the data type of the processed input supplied to the model inference function, not the data type of the source bands. Use one of the [common metadata data types](https://github.com/stac-extensions/raster?tab=readme-ov-file#data-types). |     |
+
+Note: It is common in the machine learning, computer vision, and remote sensing communities to refer to rasters that are inputs to a model as arrays or tensors. Array Objects are distinct from the JSON array type used to represent lists of values.
+
+### Model Output Object
+
+| Field Name               | Type                                          | Description                                                                                                                                                                                            |
+| ------------------------ | --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| task                     | [Task Enum](#task-enum)                       | **REQUIRED.** Specifies the Machine Learning task for which the output can be used for.                                                                                                                |
+| result_array             | [[Result Array Object](#result-array-object)] | The list of output arrays/tensors from the model.                                                                                                                                                      |
+| classification:classes   | [[Class Object](#class-object)]               | A list of class objects adhering to the [Classification extension](https://github.com/stac-extensions/classification).                                                                                 |
+| post_processing_function | string                                        | A url to the postprocessing function where normalization, rescaling, and other operations take place.. Or, instead, the function code path, for example: `my_package.my_module.my_processing_function` |
+
+
+While only `task` is a required field, all fields are recommended for supervised tasks that produce a fixed shape tensor and have output classes.
+`image-captioning`, `multi-modal`, and `generative` tasks may not return fixed shape tensors or classes.
 
 #### Task Enum
```

Comment:
To follow the same convention as `ml-model` and multiple websites using these tasks as "tags", the spaces should all be replaced by `-`.

Reply:
https://github.com/rbavery/dlm-extension/pull/2